### DISCUSSION TITLE
shift-alt-s does not work when trying to open up an app when doing skiovox

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#952] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
My version is 114.0.5735.351

**Comment by CGRmags** [comment]
@Vhirttuegaming  then i have no idea im on 120 so maybe upgrade it?

**Reply by v4pl** [reply]
Don't update do not 120 is the best version I wish I didn't update do caub cause Skiovox work for 120

**Reply by CGRmags** [reply]
hes on 114 so i was telling him to upgrade to 120 bruh

**Reply by v4pl** [reply]
Oh ya sorry that works

**Reply by CGRmags** [reply]
thats fine lol i have lots of misunderstandings sometimes to

**Reply by Vhirttuegaming** [reply]
@AshtonDavies you know a new way to solve this problem?

**Reply by v4pl** [reply]
just saying skiovox is currenty patched for 121 if your 120 check my profile for all you 121 users just use ltmeat flood

**Reply by Vhirttuegaming** [reply]
im 114 -_- and I cannot update.

**Reply by CGRmags** [reply]
@vaperbest in the titanium network skiovox kajij a person that goes "anonymous" talks about another method for skiovox with like search and period.

**Reply by CGRmags** [reply]
lol idk what im saying but someone mentions smth for skiovox for 121

**Reply by hank929** [reply]
who

**Reply by hank929** [reply]
did anyone know how to downgraded chrome version?

**Reply by S-PScripts** [reply]
> Vhirttuegaming

Can you update now? if not use a new exploit
